Diary/Controller/DiaryDetailViewController.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  DiaryDetailViewController.swift
+//  Diary
+//
+//  Created by Dasan, kyungmin on 2023/08/30.
+//
+
+import UIKit
+
+final class DiaryDetailViewController: UIViewController {
+    private let diary: Diary
+    private let diaryTextView: UITextView = {
+        let textView = UITextView()
+        textView.font = UIFont.preferredFont(forTextStyle: .body)
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return textView
+    }()
+    
+    init(diary: Diary) {
+        self.diary = diary
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
```suggestion
        super.init(coder:coder)
```
요런식으로도 풀어가는 방법이 있긴합니다~ fatalError가 찜찜할 경우에는 말이죠 ㅎㅎ

Reply:
알려주신 방법으로 적용해봤습니다😊